code/__HELPERS/_cit_helpers.dm

```diff
@@ -139,6 +139,17 @@ GLOBAL_VAR_INIT(miscreants_allowed, FALSE)
 	else
 		return "<span class='notice'>[html_encode(copytext_char(msg, 1, 37))]... <a href='?src=[REF(src)];flavor_more=1'>More...</span></a>"
 
+/mob/proc/print_flavor_text_2(flavor)
```

Comment:
Just add a second arg for the href command name to the original proc. None of this copypasta, please.